### PR TITLE
adding delegatable anonymous credentials proposal

### PR DESCRIPTION
**Pull request by dhuseby**
Signed-off-by: Dave Huseby <dave@cryptid.tech>

### CODE REVIEW

File: work_items/cbb_delegatable_anonymous_credentials.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- A white paper describing the cryptographic guarantees and applications of
+  the anonymous credentials in decentralized crypto-systems.
+- One or more reference implementations.
+- The expected timeline is 1-3 months to completion.
+- The audience is any developer that is working with authentic data and
+  applying it to fully decentralized authentication systems.
+
+### Relevant Related Standards
+- [Delegatable Anonymous Credentials from Mercurial Signatures][0]
+- [Authorization Capabilities for Linked Data][1] (or how *not* to do it)
```
Comment (OR13) [inline work_items/cbb_delegatable_anonymous_credentials.md:41]: https://github.com/w3c-ccg/zcap-ld/issues/37

Comment (OR13) [reply work_items/cbb_delegatable_anonymous_credentials.md:41]: Hopefully we can gather constructive criticism and avoid slinging mud :)

Comment (dhuseby) [reply work_items/cbb_delegatable_anonymous_credentials.md:41]: ; ) that was for you Orie. I'll remove it.

Comment (dhuseby) [reply work_items/cbb_delegatable_anonymous_credentials.md]: @OR13 I cleaned up the language to be more generic without pointing fingers.

File: work_items/cbb_delegatable_anonymous_credentials.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+decentralized crypto-systems that maximize privacy and have authorization
+functions that rely on credential delegation and revocation.
+
+This is similar in nature to the IETF JOSE and COSE standards work that
+formalized the data model and method for encoding cryptographically signed JSON
+and CBOR encoded data objects respectively.
+
+## Work Item Owners
+- Dave Huseby (@dhuseby, dave@cryptid.tech, CryptID Technologies, Inc)
+- Hopefully somebody else :)
```
Comment (OR13) [inline work_items/cbb_delegatable_anonymous_credentials.md:20]:
```suggestion
- Orie Steele (@OR13)
- Hopefully somebody else :)
```

**Review by OR13 — CHANGES_REQUESTED** [review]
I'd like to help with this.

**Comment by dhuseby** [comment]
I've tried to resolve @OR13's suggested changes and GH still doesn't see them as resolved. This is ready to merge though.

**Comment by brentzundel** [comment]
Changes requested and made, WG approved during our last meeting. Merging

**Pull request merged**